hedera-mirror-rest/viewmodel/contractViewModel.js

```diff
@@ -55,6 +55,9 @@ class ContractViewModel {
     if (contract.bytecode !== undefined) {
       this.bytecode = utils.addHexPrefix(contract.bytecode, true);
     }
+    if (contract.runtimeBytecode !== undefined) {
+      this.runtime_bytecode = utils.addHexPrefix(contract.runtimeBytecode, true);
+    }
```

Comment:
We should never conditionally show fields in the response. If the bytecode is not present it should return null instead of omit the key entirely.

```suggestion
    this.bytecode = contract.bytecode !== undefined utils.addHexPrefix(contract.bytecode) : null;
    this.runtime_bytecode = contract.runtimeBytecode !== undefined ? utils.addHexPrefix(contract.runtimeBytecode) : null;
```

Reply:
believe it's because in getContracts (for `/api/v1/contracts`) we don't select the contract's bytecode from db

Reply:
Oh, didn't realize we were using the same viewmodel for both. We should have different viewmodels with the list being a subset of the by ID, similar to how we handle contract results.

Reply:
The view models are separated now.